hub/util/tests/test_split.py

```diff
@@ -0,0 +1,18 @@
+import numpy as np
+from hub.util import split
+
+
+def test_split(memory_ds):
+    ds = memory_ds
```

Comment:
nit-pick: Can u test with a larger dataset (1000+) and assert the len of each split.

Reply:
@imshashank This won't work yet. `len(ds)` still returns the length of the total dataset, not the indexed version. This is in the backlog.